client/src/components/ElementEditor/ElementEditor.scss

```diff
@@ -1,3 +1,9 @@
 .element-editor {
   background-color:  $white;
+
+  margin-top: calc(-1rem + -1px);
```

Comment:
I realise this is largely my fault, but I think this needs a comment to describe where these magic numbers come from. The -1rem may also have a variable - but in the least the `-1px` needs to be explained as matching the border-bottom-width of the field above it.

Reply:
Sweet, will add a comment now.

Reply:
What's the computed value? It probably is a variable we can use

Reply:
Unfortunately not. 15px come from .form-group and we move up one pixel to hide the last form element's border, which doesn't' spread full width.